src/engine/strat_engine/pool.rs

```diff
@@ -423,61 +395,50 @@ impl Pool for StratPool {
     /// Precondition: All filesystems belonging to this pool must be
     /// unmounted.
     fn destroy(self) -> EngineResult<()> {
-        teardown_pool!(self);
+        let dm = try!(DM::new());
```

Comment:
Yes. My plan is to make the header comments in the definition of the Pool trait really useful. If you look at the header comment for this method in the Pool trait definition, you will see that it has gotten bad.